packages/flutter_splash_tizen/bin/create.dart

```diff
@@ -0,0 +1,36 @@
+import 'file_utils.dart';
+import 'package:xml/xml.dart';
+
+void main() {
+  var doc = loadYamlFileSync("pubspec.yaml")?['flutter_splash_tizen'];
```

Comment:
if I understand correctly, current flow supports only adding a file once, then file added to yaml will be copied to tizen-manifest. What about changing a file mentioned in yaml during the development? Shouldn't we consider a scenario that a file is added to manifest file everytime when application is built which means that the file used in tizen manifest will be kept in sync with file mentioned in pubspec.yaml file?

Reply:
@swift-kim could you please share your opinion regarding this comment? Should we set splash screen 'on demand' only, or rather do it automatically on every build?

Reply:
I think it's better to sync automatically every build if possible. By the way, is there a way to run this package every build?
Should we add this feature to `flutter-tizen` tool?

Reply:
`doc` can be `null`. please check it.

Reply:
@pkosko You can just follow flutter_native_splash's way. So manually running the pub run command every time the attributes are changed should just be enough.